mne/io/base.py

```diff
@@ -1765,6 +1769,80 @@ def to_data_frame(self, picks=None, index=None,
                                default_index=['time'])
         return df
 
+    def describe(self, data_frame=False):
+        """Describe channels (name, type, descriptive statistics).
+
+        Parameters
+        ----------
+        data_frame : bool
+            If True, return a pd.DataFrame. If False, print descriptive
+            statistics (and return None).
+        """
+        from scipy.stats import scoreatpercentile as q
+        nchan = self.info["nchan"]
+
+        # describe each channel
+        cols = defaultdict(list)
+        cols["name"] = self.ch_names
+        for i in range(nchan):
+            ch = self.info["chs"][i]
+            data = self[i][0]
+            cols["type"].append(channel_type(self.info, i))
+            cols["unit"].append(_unit2human[ch["unit"]])
+            cols["min"].append(np.min(data))
+            cols["q1"].append(q(data, 25))
+            cols["median"].append(np.median(data))
+            cols["q3"].append(q(data, 75))
+            cols["max"].append(np.max(data))
+
+        if data_frame:  # return data frame
+            import pandas as pd
+            df = pd.DataFrame(cols)
+            df.index.name = "ch"
+            return df
+
+        # convert into commonly used units
+        for i in range(nchan):
+            if cols['type'][i] == "eeg" and cols['unit'][i] == "V":
+                cols['unit'][i] = "µV"
+                scale = 1e6
+            elif cols['type'][i] == "mag" and cols['unit'][i] == "T":
+                cols['unit'][i] = "fT"
+                scale = 1e15
+            elif cols['type'][i] == "grad" and cols['unit'][i] == "T/m":
+                cols['unit'][i] = "fT/cm"
+                scale = 1e13
+            else:
+                scale = 1
+            for col in ["min", "q1", "median", "q3", "max"]:
+                cols[col][i] *= scale
```

Comment:
you should use the _handle_default function as used in mne/viz/evoked.py

my concern is that we have now much more types of data (ecog, seeg, dbs, fnirs etc.)

Reply:
I'd need to parse units then. I derive the scale factor from `DEFAULTS['units']` in `defaults.py` and you're right, I could use `_handle_default`, but the real problem is to convert `fT/cm` to `1e13`.

Reply:
Also, I've already added all units from that dict so I don't think anything is missing. But I'll take a look at how to use existing functionality!

Reply:
Done!